extensions/azurePublish/src/node/index.ts

```diff
@@ -177,84 +150,6 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
     /*******************************************************************************************************************************/
     /* These methods implement the publish actions */
     /*******************************************************************************************************************************/
-    /**
-     * Prepare a bot to be built and deployed by copying the runtime and declarative assets into a temporary folder
-     * @param project
-     * @param settings
-     * @param srcTemplate
-     * @param resourcekey
-     */
-    private init = async (project: any, srcTemplate: string, resourcekey: string, runtime: any) => {
```

Comment:
This section of code is where we set up the temporary folders for handling non-ejected bots.  BYE!